lib/actions/ui.js

```diff
@@ -279,7 +279,7 @@ export function handleBackButtonPress(e) {
     if (previousSearch) {
       // If back button pressed and active search has changed, set search to
       // previous search ID.
-      if (activeSearchId !== previousSearchId) {
+      if (activeSearchId && activeSearchId !== previousSearchId) {
```

Comment:
How does this have to do with `autoplan = false`?

Reply:
I think if the search changes, the activeSearchId changes, but if autoPlan is false, the activeSearchId still changes but is set to null, which is triggering this check.